react-client/src/PeoplePicker.css

```diff
@@ -2,9 +2,13 @@
 
   .chosenPeople,
   .unchosenPeople {
-    display: grid;
+display: flex; 
+flex-direction: row;
```

Comment:
Let's leave it as a grid. If it's a flexbox, a Person on their own row spans the entire thing or doesn't resize at all. Please restore the grid code.

Reply:
I pulled out the reference to flex specifically in the unchosen people and chosen people section. I kept it in the styling for the individual person component because I do not believe it has the same effect as mentioned above. Let me know if this is not the case.